Sources/DarkModeKit/Extensions/UISlider+DarkModeKit.swift

```diff
@@ -17,5 +17,8 @@ extension UISlider {
     if let dynamicMaximumTrackTintColor = maximumTrackTintColor?.copy() as? DynamicColor {
       maximumTrackTintColor = dynamicMaximumTrackTintColor
     }
+    if let thumbTintColor = thumbTintColor?.copy() as? DynamicColor {
```

Comment:
Maybe we can use `dynamicThumbColor` instead of `thumbTintColor ` because usage of the `self.` in this block.
```
if let dynamicThumbColor = thumbTintColor?.copy() as? DynamicColor {
    thumbTintColor = dynamicThumbColor
}
```